task22/js/main.js

```diff
@@ -128,21 +123,33 @@ const changeSortStatus = (status) => {
     }
 }
 
+const addEventForSortButton = (userContentsData) => {
+    const sortButtons = document.querySelectorAll(".js-sortButton");
+    sortButtons.forEach((button) => {
+        button.addEventListener('click', (e) => {
+            const nextStatus = changeSortStatus(e.target.dataset.status);
+            e.target.dataset.status = nextStatus;
+            button = e.currentTarget.id;
+            sortContents(button, userContentsData, nextStatus);        
```

Comment:
Q.
I think it's hard to tell that the variable name here is "button", but is there an intention to specify it again for "button"?
The following is my proposal.

```suggestion
            const targetId = e.currentTarget.id;
            sortContents(targetId, userContentsData, nextStatus);        
```

Reply:
Thank you for pointing this out. I was writing this with a sense of discomfort myself. I am glad it is a better code.
[5689b91](https://github.com/seeeen044/JavaScript_handson/pull/24/commits/5689b91fc27f280f719cf08ea7c31de4ac40dd7e)